.gitignore

```diff
@@ -48,3 +48,7 @@ client/yarn-debug.log*
 client/yarn-error.log*
 
 client/.idea/
+
+debug.log
+package-lock.json
```

Comment:
no reason to add to gitignore and store this file if you use `yarn` b/c yarn creates its own lock file named yarn.lock and I can't image a case where someone needs to use both managers to handle dependencies, so you can remove it

Reply:
done